example/spec/helpers/kinesisHelpers.js

```diff
@@ -141,10 +141,12 @@ async function getShardIterator(streamName) {
 
   const streamDetails = await kinesis.describeStream(describeStreamParams).promise();
   const shardId = streamDetails.StreamDescription.Shards[0].ShardId;
+  const startingSequenceNumber = streamDetails.StreamDescription.Shards[0].SequenceNumberRange.StartingSequenceNumber;
 
   const shardIteratorParams = {
     ShardId: shardId, /* required */
-    ShardIteratorType: 'LATEST',
+    ShardIteratorType: 'AT_SEQUENCE_NUMBER',
```

Comment:
I'm running tests now to see if I needed this change at all.  I think that I don't.

Reply:
I was wrong, after testing, we need to start at beginning I believe.

Reply:
This could probably be set to 'TRIM_HORIZON' for the same effect, but I know this works in testing.